packages/modules/web_themes/koala/source/src/components/models/table-model.ts

```diff
@@ -0,0 +1,47 @@
+import { QTableColumn } from 'quasar';
+
+// export type columnConfig = {
+//   fields: string[];
+//   fieldsExpansionRow?: string[];
+//   labels?: Record<string, string>;
+//   align?: Align;
+// };
+
+export type columnConfiguration = {
```

Comment:
Typendefinitionen bitte immer komplett in CamelCase mit großem Anfangsbuchstaben.
```suggestion
export type ColumnConfiguration = {
```